cypress/page-objects/student-path.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+
+import { contains } from "cypress/types/jquery";
+
+export class StudentPath {
+  goToSubjectPage(): void {
+    cy.get(".font-medium > .inline-flex")
+    cy.contains("Try it out").click().wait(1000);
+      }
+  checkSubjectPage(): void {
+    cy.get("h1.text-4xl").should("have.text", "Subjects page").wait(1000);
+  }
```

Comment:
I would include this assertion in the method 'goToSubjectPage' right after:
 `cy.contains("Try it out").click().wait(1000);`. No need for separate method for assertion.